jupyterlab_comments/jupyterlab_comments/git_commands.py

```diff
@@ -29,7 +31,7 @@ def __init__(self, **kwargs):
 
 	def run(self, cwd, *args):
 		try:
-		  return subprocess.check_output(['git'] + list(args), cwd=cwd)
+		  return subprocess.check_output(['git'] + list(args), cwd=cwd, stderr=subprocess.DEVNULL)
```

Comment:
Running the tests with multiple git calls was producing a lot of output that I didn't want to see every time I re-ran them -- I will take this out since it is needed for debugging the actual extension